app/components/dsfr_component/badge_component.rb

```diff
@@ -0,0 +1,30 @@
+module DsfrComponent
+  class BadgeComponent < DsfrComponent::Base
+    STATUSES = %i[success error info warning new].freeze
+
+    # @param status [Symbol] one of the BadgeComponent::STATUSES
```

Comment:
I'm guessing it would be more helpful for doc readers to hardcode this list in this comment

Reply:
I was hoping RDoc would expose the enum and do some sort of cross-reference but I haven't checked. If it doesn't I'll put the list, makes sense 👍